content/opt/run

```diff
@@ -19,6 +19,10 @@ chown -R rundeck:rundeck /var/rundeck
 chown -R rundeck:adm /var/log/rundeck
 chown -R rundeck:rundeck /var/lib/rundeck
 chown -R rundeck:rundeck /opt/rundeck-defaults
+rm -rf /tmp/rundeck
```

Comment:
@anojansivarajah 

Yes, understood about tmp being temporary, but we shouldn't assume a user would not mount it as a volume with things they don't want deleted.  How are you passing a different UID and GID?